litmus/PTX/Manual/XF-Barrier-weak.litmus

```diff
@@ -0,0 +1,21 @@
+PTX XF-Barrier-weak
+"Adapted from Figure 2 in 
+Portable Inter-workgroup Barrier Synchronisation for GPUs
+https://dl.acm.org/doi/pdf/10.1145/2983990.2984032"
+{
```

Comment:
`x=1;` is missing here

Reply:
In dartagnan, all locations are initialized to zero (unless we explicitly give another initializing value) so there is no real need to this change. However, for consistency with the other ones, I made it.